### DISCUSSION TITLE
API cannot submit listing variants

### DISCUSSION THREAD

**Discussion by Sean190828** [etsy/open-api#1027] [Q&A]
On the same day, I encountered a need to create variants through the API, but found that the official API did not provide an API for submitting variants. Is there any plan to provide an API for variant creation?

**Comment by cmunden** [comment]
You create listing variations using inventory combined with listing products, listing offerings and taxonomy node properties.

**Comment by GorgonFreeman** [comment]
I definitely don't understand this. I create one listing, it becomes one product being sold. Should we be creating 10 listings and using some kind of shared ID to link them? I would greatly appreciate an explanation of how to create a product with variations.

EDIT: I think this is what we want: [doc](https://developer.etsy.com/documentation/tutorials/listings/#listing-inventory-with-different-properties-quantities-and-prices)

EDIT: I've read that, had a go, and still have no idea what's up with it. I just want to submit an variation title and an array of values to sell under one product. I'll update if I get anywhere.

EDIT: [Updates here - spoiler, I do crack it](https://github.com/etsy/open-api/discussions/1067)
